spec/jobs/reports/drop_off_report_spec.rb

```diff
@@ -7,11 +7,77 @@
     JSON.parse '[{"emails":["ursula@example.com"],
        "issuers":["urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name"]}]'
   end
+  let(:empty_emailable_report) do
+    report = Reporting::DropOffReport.new(
+      issuers: 'urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name',
```

Comment:
in the underlying report, issuers is expected to be an array. I would keep that consistent here
```suggestion
      issuers: ['urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name'],
```